components/ui/DocsPagination.tsx

```diff
@@ -14,109 +14,41 @@ interface PaginationProps {
 
 export function DocsPagination({ prevPage, nextPage }: PaginationProps) {
   return (
-    <Wrapper>
+    <div className="mt-8 grid grid-cols-2 gap-4">
       {prevPage && prevPage.slug && (
         <DynamicLink href={`${prevPage.slug}`} passHref>
-          <PaginationLink previous>
-            <span>Previous</span>
-            <h5>{prevPage.title}</h5>
-            <RightArrowSvg />
-          </PaginationLink>
+          <a
+            className="block p-4 text-left relative transition-all group border border-gray-100"
+            style={{ backgroundColor: '#FAFAFA' }}
+          >
+            <span className="text-sm uppercase opacity-50 pl-10">Previous</span>
+            <h5
+              className="text-xl leading-[1.3] m-0 pl transition-all ease-out duration-150 text-blue-800 group-hover:text-orange-500 flex items-center"
+            >
+              <RightArrowSvg className="w-8 h-8 fill-gray-400 transition-all ease-out duration-150 rotate-180 group-hover:fill-orange-500 mr-2" />
+              {prevPage.title}
+            </h5>
+          </a>
```

Comment:
Shouldn't we use `<Link>` here?

Reply:
Usually yes, However we dont use navigation in our <a> tags, we do so in our <DynamicLink>. Rather ive replaced our <a> with <div>'s